app/controllers/UserController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Phalcon\Paginator\Adapter\Model as PaginatorModel;
+
+class UserController extends BaseController{   
+
+    public function listAction(){
+        // Current page to show
+        $currentPage = 1;
+        $currentPage = $this->request->getQuery('page', 'int');
```

Comment:
Third parameter is optional and can be set to default number. In this case it should be 1.